### DISCUSSION TITLE
有没有交流群 (Is there any chat group?)

### DISCUSSION THREAD

**Discussion by ofearn** [qber-soft/Ave-Nodejs#64] [Q&A] [answered]
如题

>Is there any chat group?

**Comment by rerender2021** [comment, marked as answer]
有了交流群了，github上我们使用这个：https://github.com/qber-soft/Ave-Nodejs/discussions

> Yes, we prefer https://github.com/qber-soft/Ave-Nodejs/discussions in github

好处是各个问题可以单独追踪，也方便搜索

> In this way, we can track and search problems easily

即时通讯使用：

> Instant Messaging：

- qq：326315711 or https://jq.qq.com/?_wv=1027&k=6t1BwW4q
